documentation/architecture/DEPLOYMENT_GUIDE.md

```
@@ -0,0 +1,327 @@
+kubectl config current-context
+
+# 3. Test connectivity
+kubectl cluster-info
+
+# 4. If still failing, may need SSH tunnel through bastion to EKS endpoint
+# (private endpoint not directly reachable from outside VPC)
+```
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)